QKSMS/src/main/java/org/w3c/dom/smil/ElementExclusiveTimeContainer.java

```diff
@@ -31,7 +31,7 @@ public void setEndSync(String endSync)
                                      throws DOMException;
 
     /**
-     *  This should support another method to get the ordered collection of 
+     *  This should support another method to getConversation the ordered collection of
```

Comment:
Ah didn't notice that. Reject the pull and I'll fix that up
On Dec 3, 2015 7:34 PM, "Moez Bhatti" <notifications@github.com> wrote:

> In QKSMS/src/main/java/org/w3c/dom/smil/ElementExclusiveTimeContainer.java
> <https://github.com/qklabs/qksms/pull/294#discussion_r46634523>:
>
> > @@ -31,7 +31,7 @@ public void setEndSync(String endSync)
> >                                       throws DOMException;
> >
> >      /**
> > -     *  This should support another method to get the ordered collection of
> > +     *  This should support another method to getConversation the ordered collection of
>
> Seems to be a lot of comments with the word "get" that got replaced with
> "getConversation"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qklabs/qksms/pull/294/files#r46634523>.
>